.eslintrc.js

```diff
@@ -187,7 +188,8 @@ module.exports = {
     "one-var-declaration-per-line": [2, "initializations"],
     "operator-assignment": [2, "always"],
     "operator-linebreak": [2, "after"],
-    "padded-blocks": [2, "never"],
+    // padded-blocks conflicts with lines-around-comment.
+    "padded-blocks": "off",
```

Comment:
With `4.9.0` instead of turning this off, we can do this instead:
`"lines-around-comment": [2, {"allowClassStart": true, "allowObjectStart": true, "beforeBlockComment": true}],`